pkg/cmd/sync/sync.go

```diff
@@ -71,12 +67,23 @@ func (cmd *SyncCmd) Run(f *cmdutil.Factory) error {
 		}
 	}
 
+	originIds := make(map[string]contracts.AzionJsonDataOrigin)
+	for _, itemOrigin := range conf.Origin {
```

Comment:
Don't you need to do a loop for cacheIds as well?

Reply:
good point :smile: